app/views/drivers/index.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1>Rideshare Drivers</h1>
+
+<% @drivers.each do |driver| %>
+
+  <ul >
+    <li> <img src="https://via.placeholder.com/50x50"  /> <%= link_to driver.name, show_driver_path(driver[:id]) %>  <%= link_to "Edit", edit_driver_path(driver[:id]) %>  <%= link_to "Delete", delete_driver_path(driver[:id]), method: :delete %></li>
```

Comment:
It's human nature to want to click on the driver's image as well as the name, so making that a link would be nice.